tools/test/before_script.sh

```diff
@@ -148,6 +148,10 @@ elif [ $RDBMS == "sqlite" ]; then
     RESTART_HTTPD=1
 fi
 
+# external metadata providers
+docker $CONTAINER_RUNTIME_ARGS run -d $CONTAINER_RUN_ARGS docker.io/mongo:5.0
```

Comment:
This would always start a new mongodb container, please take a look at the comment I've left on this PR on making it specific to particular test cases ;)